build/mac/BUILD.gn

```diff
@@ -365,6 +375,32 @@ action("create_pkg") {
   outputs = [ unsigned_pkg_path ]
 
   deps = [ ":finalize_app" ]
+
+  if (brave_enable_updater) {
+    deps += [ ":postinstall" ]
+  }
+}
+
+action("postinstall") {
+  visibility = [ ":create_pkg" ]
+
+  script = "create_postinstall.py"
+
+  output = "$generated_pkg_scripts/postinstall"
+
+  args = [
+    rebase_path("//chrome/installer/mac/pkg_postinstall.in"),
+    brave_exe,
+    "Contents/Frameworks/$chrome_product_full_name Framework.framework",
+    rebase_path(output),
+  ]
+
+  inputs = [
+    script,
```

Comment:
should `brave_exe` be in inputs?

Reply:
I don't think so. The generated file `postinstall` does not change when `brave_exe` changes. But maybe I'm missing something?